scheduler/scheduler.cpp

```diff
@@ -105,6 +105,7 @@ static list<CompileServer *> css;
 static list<CompileServer *> monitors;
 static list<CompileServer *> controls;
 static list<string> block_css;
+static list<CompileServer *> css_temp_hold;
```

Comment:
css_temp_hold isn't a helpful name either.  It is taking me a while to figure out what it does.